requirements/edx/base.txt

```diff
@@ -111,6 +111,7 @@ edx-search==1.4.1         # via -r requirements/edx/base.in
 edx-sga==0.11.1           # via -c requirements/edx/../constraints.txt, -r requirements/edx/base.in
 edx-submissions==3.2.2    # via -r requirements/edx/base.in, ora2
 edx-tincan-py35==0.0.9    # via edx-enterprise
+git+https://github.com/regisb/edx-toggles.git@regisb/extract-waffle-utils  # via -r requirements/edx/base.in
```

Comment:
Do you want to add `edx-toggles<2.0.0` to constraints.txt in preparation for upcoming major version backward compatibility issue that we are likely to introduce when we remove name spacing?

Reply:
Yes, that could make sense.

Reply:
Ok.  You can add it once you switch to the real edx-toggles later on.